centreon/packages/ui/src/api/customFetch.ts

```diff
@@ -82,6 +82,15 @@ export const customFetch = <T>({
             };
           }
 
+          if (equals(response.status, 207)) {
+            return {
+              additionalInformation: data.results,
+              isError: false,
+              message: '',
+              statusCode: response.status
+            };
+          }
```

Comment:
maybe add some doc on this esotheric `207` status ?

why is the `data.results` mapped to `additionalInformation` instead of default `data` ?